tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2530,6 +2534,30 @@ public function userListsTheResourcesSharedWithThemUsingGraphApi(string $user):
 		);
 	}
 
+	/**
+	 * @When user :user lists the shares shared with him/her using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userListsTheResourcesSharedWithThemUsingGraphApi(string $user): void {
+		$this->getSharesSharedWithMe($user);
```

Comment:
Would it be better if the response is set with the variable in the `getSharesSharedWithMe`  and then these steps called that variable and set the response here? IMO that would be consistent with our code.